server/src/WebSocketEvents.js

```diff
@@ -763,6 +763,20 @@ eventResponses.start_dungeon = (clientSocket, data) => {
     dungeonPortal.enter(player);
 };
 
+/**
+ * @param {*} clientSocket
+ * @param {String} data.dungeonID - The ID of a dungeon manager.
```

Comment:
This line of param docs for "data.dungeonID" isn't used so can be removed, as only `clientSocket` is used in the callback function.